jbpm-case-mgmt/jbpm-case-mgmt-impl/src/main/java/org/jbpm/casemgmt/impl/command/StartCaseCommand.java

```diff
@@ -125,6 +123,7 @@ public void matchCreated(MatchCreatedEvent event) {
         }
         // set case id to allow it to use CaseContext when creating runtime engine
         params.put(EnvironmentName.CASE_ID, caseId);
+        params.put(VariableScope.FORCE_UPDATE, Boolean.TRUE);
```

Comment:
I am not very convince about this.
If you are starting a case o restarting one to avoid any problem with variable you just need to check whether the oldValue and the newOne are the same instead of this. This will fix the issue regarding using the newCaseFileInstance + startCase... reopening will be more tricky as drools and facts are involved.

Reply:
Additional parameter has been removed. 
assuming assigment of equal object is not considered assigment, we can avoid it